src/System.Private.Xml/tests/Readers/ReaderSettings/MaxSettings.cs

```diff
@@ -12,7 +12,7 @@ namespace System.Xml.Tests
     [TestCase(Name = "MaxCharacters Settings", Desc = "MaxCharacters Settings")]
     public partial class TCMaxSettings : TCXMLReaderBaseGeneral
     {
-        private long _defaultCharsEnt = (long)1e7;  // By default, entity resolving is limited to 10 million characters (On full .NET the default used to be zero (=unlimited) as LegacyXmlSettings was enabled)
+        private long _defaultCharsEnt = (PlatformDetection.IsFullFramework)?0:(long)1e7;  // By default, entity resolving is limited to 10 million characters (On full .NET the default used to be zero (=unlimited) as LegacyXmlSettings was enabled)
```

Comment:
please check the quirk value when checking for full framework, otherwise we will get this failing once we change netfx version we target